app.R

```diff
@@ -347,7 +347,11 @@ server <- function(input, output, session) {
   filtered_race_results <- reactive(
     race_results |> 
       dplyr::filter(GP == input$gp) |> 
-      dplyr::select(-GP) 
+      dplyr::select(-GP, -Track)  |> 
+      dplyr::mutate(Position = as.integer(Position)) |> 
+      dplyr::select(Driver, Team, Position, `Time/Retired`, Laps,
+                    `Starting Grid`, Points, `Fastest Lap`, flag) 
+    
   )
```

Comment:
Add 'number' column between 'driver' and 'team' columns. (This is the number that belongs to the driver)

Reply:
Done

Reply:
Did you push this? Can't see the number column yet